enterprise_gateway/services/kernels/handlers.py

```diff
@@ -146,11 +153,178 @@ def get(self, kernel_id):
         self.finish(json.dumps(model, default=date_default))
 
 
-default_handlers = []
+class ConfigureMagicHandler(CORSMixin, JSONErrorsMixin, APIHandler):
+    @web.authenticated
+    async def post(self, kernel_id):
+        self.log.info(f"Update request received for kernel: {kernel_id}")
+        km = self.kernel_manager
+        km.check_kernel_id(kernel_id)
+        payload = self.get_json_body()
+        self.log.debug(f"Request payload: {payload}")
+        if payload is None:
+            self.log.info("Empty payload in the request body.")
+            self.finish(
+                json.dumps(
+                    {"message": "Empty payload received. No operation performed on the Kernel."},
+                    default=date_default,
+                )
+            )
+            return
+        if type(payload) != dict:
+            self.log.info("Payload is not in acceptable format.")
+            raise web.HTTPError(400, "Invalid JSON payload received.")
```

Comment:
Same goes for these `info` messages that precede exceptions.  I believe EG will be logging these exceptions.

Could you please append something like `f" for kernel: {kernel_id}"` to the end?  As EG becomes more multi-tenant, its important we try to include some unique identifier in messages wherever possible.